RecoTracker/LSTCore/src/alpaka/TrackCandidate.h

```diff
@@ -167,8 +167,8 @@ namespace SDL {
                                                     struct SDL::miniDoublets& mdsInGPU,
                                                     struct SDL::segments& segmentsInGPU,
                                                     struct SDL::hits& hitsInGPU) {
-    int phits1[4] = {-1, -1, -1, -1};
-    int phits2[4] = {-1, -1, -1, -1};
+    int phits1[hits_pLS] = {-1, -1, -1, -1};
+    int phits2[hits_pLS] = {-1, -1, -1, -1};
```

Comment:
```suggestion
    int phits2[hits_pLS];
```
initialization isn't really necessary